version.in

```diff
@@ -0,0 +1,4 @@
+#define LC0_VERSION_MAJOR @major@
```

Comment:
Looks like you renamed this file by accident?

Reply:
No, this is a standard for templated files.

Reply:
oh my bad, right.